lib/curl_sasl.c

```diff
@@ -58,7 +58,7 @@
 static const struct {
   const char   *name;  /* Name */
   size_t        len;   /* Name length */
-  unsigned int  bit;   /* Flag bit */
+  unsigned long bit;   /* Flag bit */
```

Comment:
Why this change? We cannot assume that a long is larger than 32 bit anyway... and if you want to store 32 bit, why use 64 bit on some platforms?

Reply:
HTTP authentication flags are already defined as unsigned long: although not yet related, the idea behind this change is to ease a possible future unification of sasl and http authentication flags.
This is the only reason. But perhaps the change would have to be done the other way on http side, which is not the target of my current work.
I can drop it for now, if you prefer.